### DISCUSSION TITLE
Dataverse Teams commands

### DISCUSSION THREAD

**Discussion by Laskewitz** [microsoft/powerplatform-build-tools#473] [Ideas]
# Idea

I would love to see commands in CLI to work with Dataverse teams. Being able to create a Dataverse team based on the following team types:

- Microsoft Entra Identity Security Group
- Microsoft Entra Identity Office Group
- Owner
- Access

# Commands

`pac team create` - Create a team (is basically what `pac admin assign-group` already does)
`pac team delete` - Delete a team
`pac team update` - Update a team (name, description, business unit)
`pac team list` - List teams
`pac team add-user` - Add a user to a team
`pac admin assign-team` - Add a security role to a team

# Remarks

This would be very helpful in setup scenarios, for instance when you want to add a team and assign one of the pipelines roles, that are available when you install the pipelines solution on an environment.

**Comment by anttipajunen** [comment]
At least AAD Group Teams (I guess these should now be called Entra ID Group Teams) would be pretty handy to create with PAC CLI.

**Comment by mcsmith3** [comment]
Has there been any further discussion on this? I would love the ability to add teams easily with an iterative PowerShell script and then assign users to it.
